dbms/src/Server/Server.cpp

```diff
@@ -1167,8 +1167,6 @@ try
             }
         }
 
-        SCOPE_EXIT({ proxy_machine.stopProxy(tmt_context); });
```

Comment:
There is a FlashGrpcServerHolder which depends on a DiagnosticsService which will call proxy. So I think we keep put it here.

Reply:
I have changed to destroy `FlashGrpcServerHolder` first, and then calling `stopProxy`.